.github/workflows/update-content-scope.yml

```diff
@@ -87,12 +87,12 @@ jobs:
             Content scope scripts have been updated and a PR created.
 
             If tests failed check out https://app.asana.com/0/1202561462274611/1203986899650836/f for further information on what to do next.
-
+ d
             See ${{ steps.create-pr.outputs.pull-request-url }}
           action: 'create-asana-task'
 
       - name: Add Asana task to Release Board project
-        if: ${{ steps.update-check.outcome == 'failure' }}
+        if: ${{ steps.create-task.outputs.duplicate == 'false' }}
         uses: duckduckgo/native-github-asana-sync@v1.0
```

Comment:
Any reason why we don't use `v1.1` everywhere? (here and in a few other places below)